Makefile

```diff
@@ -36,7 +36,7 @@ generate: keyservice/keyservice.pb.go
 	$(GO) generate
 
 %.pb.go: %.proto
-	protoc --go_out=plugins=grpc:. $<
+	protoc --go_out=. --go-grpc_out=require_unimplemented_servers=false:.  $<
```

Comment:
This should be the current way of generating files with protoc since 1.20 I believe.

The generated files are also split now as you can see below.

The flag `require_unimplemented_servers=false` is necessary to preserve the previous functionality.

Please note that I am using: 
![image](https://github.com/mozilla/sops/assets/29378614/64596de0-0293-4b5f-9429-7b12867d9f3c)

I have also tried 
```
protoc-gen-go v1.23.0
protoc        v3.13.0
```
but I am still getting errors with the plugin settings.

I don't think this is an issue but just flagging for the reviewer